app/main/posts/views/post-view-data.directive.js

```diff
@@ -244,23 +244,20 @@ function PostViewDataController(
 
     function confirmEditingExit() {
         var deferred = $q.defer();
-        if (!$scope.editMode.editing) {
+        if ($state.$current.name !== 'posts.data.edit') {
```

Comment:
Hey look `$state` already knows where we are!

Reply:
yay. Hate hate hate the editmode checks